### DISCUSSION TITLE
log4j vulnerabilities

### DISCUSSION THREAD

**Discussion by williamsjoehenry** [VSCodeVim/Vim#8001] [Q&A] [answered]
Hi everyone,

My apologies. I know this is kind of a goofy question. My job requires that I positively verify that any VSCode extensions I download do not contain any log4j vulnerabilities. As far as I know, this extension makes no use of log4j. Is that correct?

Thank you!

**Comment by J-Fields** [comment, marked as answer]
That's correct, VSCodeVim doesn't use log4j. It's written entirely in typescript, so I'm not sure using log4j is really even an option.
